### DISCUSSION TITLE
How to install prefect 0.15.3 in centos offline (on server without internet)

### DISCUSSION THREAD

**Discussion by arunajakhmola** [PrefectHQ/prefect#4992] [Q&A]
Hello Friends 

I need to install prefect 0.15.3 in centos 7.6 in offline mode (server has no internet connectivity).
Please guide me.

Thanks & regards,
Aruna

**Comment by pikhovkin** [comment]
linked with #4370

**Comment by zanieb** [comment]
Hi @arunajakhmola, you can download the wheel from https://pypi.org/project/prefect/#files transfer it to your server and install it with pip.
